tests/test_workflow_dicom_reorg.py

```diff
@@ -467,17 +467,17 @@ def test_cleanup_curation_status(
         (
             0,
             1,
-            f"[{LogColor.FAILURE}]Reorganized files for {{0}} out of {{1}} participant-session pairs",  # noqa: E501
+            "Reorganized files for {0} out of {1} participant-session pairs",  # noqa: E501
         ),
         (
             1,
             2,
-            f"[{LogColor.PARTIAL_SUCCESS}]Reorganized files for {{0}} out of {{1}} participant-session pairs",  # noqa: E501
+            "Reorganized files for {0} out of {1} participant-session pairs",  # noqa: E501
         ),
         (
             2,
             2,
-            f"[{LogColor.SUCCESS}]Successfully reorganized files for {{0}} out of {{1}} participant-session pairs",  # noqa: E501
+            "Reorganized files for {0} out of {1} participant-session pairs",  # noqa: E501
```

Comment:
Don't need the `noqa` comments anymore

```suggestion
            "Reorganized files for {0} out of {1} participant-session pairs",
       ),
       (
           1,
           2,
            "Reorganized files for {0} out of {1} participant-session pairs",
       ),
       (
           2,
           2,
            "Reorganized files for {0} out of {1} participant-session pairs",
```